### PR TITLE
[MIG] mail_inline_css: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**
Depends on:
- `email_template_qweb` migrated in https://github.com/OCA/social/pull/679

### CODE REVIEW

**Review by simahawk — APPROVED** [review]
@phuctranerp can you drop the tmp dependency?

**Comment by phuctranfxvn** [comment]
@simahawk, updated, thanks

File: mail_inline_css/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "Mail Inline CSS",
     "summary": "Convert style tags in inline style in your mails",
-    "version": "13.0.1.0.2",
+    "version": "14.0.1.0.2",
```
Comment (sebalix) [inline mail_inline_css/__manifest__.py]: Should we not reset to `14.0.1.0.0`?

Comment (simahawk) [reply mail_inline_css/__manifest__.py]: @phuctranerp can you fix it?

Comment (simahawk) [reply mail_inline_css/__manifest__.py]: @phuctranerp @dzungtran89 can you fix it?

Comment (dzungtran89) [reply mail_inline_css/__manifest__.py]: hi @simahawk we'll update it

Comment (hailangvn) [reply mail_inline_css/__manifest__.py]: @simahawk , the PR is updated. Thanks.

**Pull request merged**